frontend/src/pages/Profile/Admin/admin.tsx

```diff
@@ -229,7 +229,46 @@ const Column = ({
             /></View>
       </View> */
 }
+//User Crude
+const createUser = (userData: User) => {
```

Comment:
this method needs to be declared within the UsersTableHeader component, not outside of it